### PR TITLE
Added a CTA for Drop-ins in the home page

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

**Review by kellymore — APPROVED** [review]

File: app/components/base/Card.tsx [hunk 1]
```diff
@@ -148,4 +107,72 @@ const DescriptionStyle = styled.div`
...
+        <Badge text={badge} />
+      </BadgeStyle>
+    )}
+
+    {link && link.external && (
+      <LinkIconStyle>
+        <NewTabIcon width={13} />
+      </LinkIconStyle>
+    )}
+    <ConditionalWrapper
```
Comment (ShreyaThapa) [inline app/components/base/Card.tsx:147]: Added a ConditionalWrapper Util component that wraps elements based on a condition. In this case, the `Topic` and `Description` are wrapped in a `<div>` when "isFlex" is present.

File: app/components/base/__tests__/Card.test.tsx [hunk 2]
```diff
@@ -114,3 +114,44 @@ test("with Center Align, Link and Badge", () => {
...
+        badge="BADGETEXT"
+        icon={icon}
+        topic="topic"
+        description="Description"
+      />
+    )
+    .toJSON();
+
+  expect(tree).toMatchSnapshot();
+});
```
Comment (ShreyaThapa) [inline app/components/base/__tests__/Card.test.tsx:157]: Added test cases for new variant in the Card component.

File: app/components/util/ConditionalWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Used for conditionally wrapping elements
+
+const ConditionalWrapper = ({ condition, wrapper, children }) =>
+  condition ? wrapper(children) : children;
+
+export default ConditionalWrapper;
```
Comment (ShreyaThapa) [inline app/components/util/ConditionalWrapper.tsx:6]: Conditional wrapper util function

File: pages/index.mdx [hunk 4]
```diff
@@ -207,6 +207,7 @@ export default function HomePage() {
         </CardGridWrapper>
         <SingleDivWrapper>
           <Card
+            isFlex
```
Comment (ShreyaThapa) [inline pages/index.mdx:210]: Added a variant for a horizontal Card with flexed elements

File: stories/base/Card.stories.tsx [hunk 5]
```diff
@@ -87,4 +87,32 @@ storiesOf("base|Card", module)
...
+  ))
+  .add("`isFlex` + badge + link", () => (
+    <Card
+      isFlex
+      link={{ href: "https://www.dwolla.com", external: true }}
+      icon={guideIcon}
+      badge="BADGETEXT"
+      topic="This is the Topic."
+      description="This is a  description.An existing unverified bank can be verified with microdeposits. Send two small debits and have your Customer verify the amounts."
+    />
```
Comment (ShreyaThapa) [inline stories/base/Card.stories.tsx:117]: Added Stories that use the new Card variant

**Pull request merged**